server/src/main/java/org/elasticsearch/synonyms/SynonymsManagementAPIService.java

```diff
@@ -235,7 +274,7 @@ public void putSynonymsSet(
                         ? UpdateSynonymsResultStatus.CREATED
                         : UpdateSynonymsResultStatus.UPDATED;
 
-                    reloadAnalyzers(resourceName, bulkInsertResponseListener, updateSynonymsResultStatus);
+                    reloadAnalyzers(synonymSetId, bulkInsertResponseListener, updateSynonymsResultStatus);
```

Comment:
I took the opportunity to rename `resourceName` to `synonymSetId` in several places - sorry about that.

Reply:
I like the clarification